src/dynamic_bridge.cpp

```diff
@@ -254,11 +254,14 @@ void update_bridge(
     bridge.ros1_type_name = ros1_type_name;
     bridge.ros2_type_name = ros2_type_name;
 
+    bool ros1_pub_latching = (topic_name == "/tf_static");
```

Comment:
```suggestion
    auto ros2_subscriber_qos = rclcpp::QoS(rclcpp::KeepLast(10));
    bool ros1_pub_latching = false;
    if (topic_name == "/tf_static") {
      ros1_pub_latching = true;
      ros2_subscriber_qos.keep_all();
      ros2_subscriber_qos.transient_local();
    }
```
I think there should also be ros2_subscriber_qos set here to `transient_local` since the ros2 subscriber might not come alive until after the /tf_static topic has been puslihed to and so will miss the message. This fixes it.